MTAirCompressor/v1/_init.yaml

```diff
@@ -0,0 +1,6 @@
+instances:
+  -
+    sal_index: 1
+  -
+    sal_index: 2
```

Comment:
It seems odd to have just this, with no instance-specific configuration.

I would expect to see at least the host name as an instance-specific field.

If you truly need nothing then perhaps it doesn't need to be a configurable CSC.

Reply:
Host is optional, by default it is m1m3cam-aircomp0<index>.cp.lsst.org. But I will put down grace period, which we might want to change